eliot-service/tests/test_cache_manager.py

```diff
@@ -2,7 +2,9 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at https://mozilla.org/MPL/2.0/.
 
+import json
 import pathlib
+from unittest.mock import ANY
```

Comment:
`ANY` is a magic amazing thing. Prior to discovering it existed, I wrote a `WHATEVER` that was kind of the same thing, but funnier to read.